packages/server/tests/acceptance/hbarLimiter.spec.ts

```diff
@@ -887,5 +914,77 @@ describe('@hbarlimiter HBAR Limiter Acceptance Tests', function () {
         });
       });
     });
+
+    describe('@hbarlimiter-batch3 BASIC Tier', () => {
+      const accounts: AliasAccount[] = [];
+
+      beforeEach(async function () {
+        logger.info(`${requestDetails.formattedRequestId} Creating accounts`);
+        logger.info(
+          `${requestDetails.formattedRequestId} HBAR_RATE_LIMIT_TINYBAR: ${ConfigService.get(
+            'HBAR_RATE_LIMIT_TINYBAR',
+          )}`,
+        );
+
+        const initialAccount: AliasAccount = global.accounts[0];
+
+        const neededAccounts: number = 3;
+        accounts.push(
+          ...(await Utils.createMultipleAliasAccounts(
+            mirrorNode,
+            initialAccount,
+            neededAccounts,
+            initialBalance,
+            requestDetails,
+          )),
+        );
+        global.accounts.push(...accounts);
+        const basicPlans = await hbarSpendingPlanRepository.findAllActiveBySubscriptionTier(
+          [SubscriptionTier.BASIC],
+          requestDetails,
+        );
+        for (const plan of basicPlans) {
+          await hbarSpendingPlanRepository.delete(plan.id, requestDetails);
+          await ethAddressSpendingPlanRepository.deleteAllByPlanId(plan.id, 'before', requestDetails);
+          await ipSpendingPlanRepository.deleteAllByPlanId(plan.id, 'before', requestDetails);
+        }
+      });
```

Comment:
Correct me if I'm wrong but the plans might not need to be reset. It was for something special tested in batch1 BASIC tier.

Reply:
You are correct. Initially, I observed some instability when running the tests. To address this, I thought that clearing out the repositories—so that the spending plans would be recreated during the contract deployments—would provide a deterministic solution. However, after removing the code that cleared the repositories and running the tests multiple times, I achieved consistent results, indicating that the instability was unrelated to the repository state.